### PR TITLE
feat: rest infrastructure array handling

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

**Comment by rahulguptajss** [comment]
Example:
```
volume_size_available{datacenter="F2240-127-26",cluster="ocum-infinity",svm="vs5_nfs_cifs",aggr="aggr1,aggr2,aggr3,aggr4",style="flexgroup",volume="FlexGroup_Infinity",node=""} 736773632000
```

File: cmd/collectors/rest/rest.go [hunk 1]
```diff
@@ -295,7 +295,16 @@ func (r *Rest) PollData() (*matrix.Matrix, error) {
...
 			value := instanceData.Get(label)
 			if value.Exists() {
-				instance.SetLabel(display, value.String())
+				if value.IsArray() {
+					var labelArray []string
+					for _, r := range value.Array() {
+						labelString := r.String()
+						labelArray = append(labelArray, labelString)
+					}
+					instance.SetLabel(display, strings.Join(labelArray[:], ","))
```
Comment (cgrinds) [inline cmd/collectors/rest/rest.go:304]: minor but no need to copy labelArray before calling Join

Comment (rahulguptajss) [reply cmd/collectors/rest/rest.go:304]: where is the copy?

Comment (rahulguptajss) [reply cmd/collectors/rest/rest.go]: corrected.

**Review by cgrinds — APPROVED** [review]

**Pull request merged**